### ISSUE TITLE
Tables documentation page: table pagination display is broken

### ISSUE THREAD

**Issue by marteki** [cfpb/design-manual#617]
[On the documentation page for Tables, the table pagination example doesn't appear properly](https://cfpb.github.io/design-manual/page-components/tables.html#filtering-tables): the "Newer" and "Older" buttons don't scale, and they instead overlap the "Page" label and the "Go" button.

I think this is less a problem with the actual pagination component, and more an issue with the way the Design Manual page is laid out. I believe that the wider desktop view of the table is being shown because the whole page is wider, even though the table only fills up a column on the page. Is there a way we could show the table pagination more as it's intended to be seen on a production web page?

![screen shot 2018-11-06 at 11 23 51 am](https://user-images.githubusercontent.com/1183435/48079143-bc0cfd00-e1b8-11e8-9d7b-9ff764d02080.png)

**Comment by cfarm** [comment]
do you have an example of this in production?

**Comment by anselmbradford** [comment]
The only filterable table I know of is https://www.consumerfinance.gov/activity-log/, which does not have this issue.

**Comment by jimmynotjim** [comment]
> I think this is less a problem with the actual pagination component, and more an issue with the way the Design Manual page is laid out

Yep, you're right on the money. Only solution I have found is to put the table inside an iframe (this is what storybook does).
